zilliqa/src/message.rs

```diff
@@ -350,18 +350,24 @@ impl Display for QuorumCertificate {
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct AggregateQc {
     pub signature: NodeSignature,
-    pub signers: Vec<u16>,
+    pub signers: BitVec,
     pub view: u64,
     pub qcs: Vec<QuorumCertificate>,
 }
 
 impl AggregateQc {
     pub fn compute_hash(&self) -> Hash {
+        let mut signers = Vec::new();
+        for (index, bit) in self.signers.iter().enumerate() {
```

Comment:
Why not just calculate the signature over the bit vector? Why do we need to convert it to an index list?